apps/api/src/services/InsightProjectService.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { IInsightProject } from '@innbyggerpanelet/api-interfaces';
+import { Connection } from 'typeorm';
+import { InsightProject } from '../models/insightProject/InsightProjectEntity';
+import { InsightProjectService } from '../services/InsightProjectService';
+import { getTestDatabase } from '../utils/test/databaseConnection';
+import { clearDatabaseEntityTable } from '../utils/test/clearDatabaseEntityTable';
+import { createDummyInsightProject, deleteDummyInsightProject } from '../utils/test/seed/insightProject';
+import { plainToInstance } from 'class-transformer';
+import { Employee } from '../models/employee/EmployeeEntity';
+import { createDummyEmployee } from '../utils/test/seed/employee';
+
+let db: Connection;
+let insightProjectService: InsightProjectService;
+let insightProject: InsightProject;
+let seedDTO: IInsightProject;
+let employee: Employee;
+
+beforeAll(async () => {
+    db = await getTestDatabase();
+    insightProject = await createDummyInsightProject(db);
+
+    seedDTO = {
+        id: Math.floor(Math.random() * 100_000),
+        name: 'Insight project 2000',
+        description: 'The best insight project god has ever seen',
```

Comment:
The insight project to end all insight projects :P

Reply:
Where is this used btw? Seems like it is the same insight project defined in the seed file.